zds/member/forms.py

```diff
@@ -219,6 +219,7 @@ class ProfileForm(MiniProfileForm):
         ("show_markdown_help", _("Afficher l'aide Markdown dans l'éditeur")),
         ("email_for_answer", _("Recevoir un courriel lors d'une réponse à un message privé")),
         ("email_for_new_mp", _("Recevoir un courriel lors de la réception d'un nouveau message privé")),
+        ("hide_forum_activity", _("Masquer mes activités de forum et commentaires sur mon profil")),
```

Comment:
Je pense qu'il faut préciser que le masquage s'effectue pour tout le monde sauf soi-même et l'équipe de modération. Sinon, les membres vont regarder leur profil, voir que c'est encore présent et croire que ça ne fonctionne pas alors que ça fonctionne.
```suggestion
        ("hide_forum_activity", _("Masquer mes activités de forum et commentaires sur mon profil (sauf pour moi et l'équipe de modération)")),
```